book/website/community-handbook/community-calls.md

```diff
@@ -61,3 +61,19 @@ We want to make collaborative events like these as useful as possible for as man
 Please [let the team members know](https://github.com/the-turing-way/the-turing-way#get-in-touch) if there are other ways we can make these calls more welcoming and helpful for you.
 
 More templates related to Community Calls are provided in the {ref}`community template collection<ch-templates>`.
+
+### Joining Community Calls in Low-Bandwidth Environments
+
+We understand that accessing online community calls can be challenging for individuals in low-bandwidth environments.
+
+Here are some tips and alternatives for joining our calls:
+
+- **Using Zoom Mobile App:** If you're experiencing low bandwidth on your computer, consider joining the call using the Zoom mobile app on your smartphone. The app may provide better connectivity and audio quality.
```

Comment:
```suggestion
- **Using Zoom Mobile App:** If you're experiencing low bandwidth on your computer (which could be due to high usage by multiple devices, and/or slow network speeds), consider joining the call using the Zoom mobile app on your smartphone. The app may provide better connectivity and audio quality.
```

Reply:
I think there might be two different issues here: 
1) Using a mobile _network_ might be better than your wifi
2) The mobile app(s) might be better optimised for low-bandwidth situations than the desktop app(s)